Neos.ContentRepository.Core/Classes/Feature/ContentStreamCommandHandler.php

```diff
@@ -100,7 +100,7 @@ private function handleForkContentStream(
                     $sourceContentStreamVersion->unwrap(),
                 ),
             ),
-            ExpectedVersion::ANY()
+            ExpectedVersion::NO_STREAM()
```

Comment:
so this ensures that the "fork" happens as the first event of the new content stream. Makes sense I think :)